makefile

```diff
@@ -42,7 +42,7 @@ LIBTEST_S=$(LIBTEST)
 
 #List of objects to compile.
 #START_INS
-OBJECTS=src/ciphers/aes/aes_enc.o src/ciphers/aes/aes.o src/ciphers/anubis.o src/ciphers/blowfish.o \
+OBJECTS=src/ciphers/aes/aes.o src/ciphers/aes/aes_enc.o src/ciphers/anubis.o src/ciphers/blowfish.o \
```

Comment:
srsly, the new sorting sorts alphabetically in a different order than the previous sorting? :rofl: